next-tavla/app/(admin)/components/CreateBoard/index.tsx

```diff
@@ -137,21 +151,28 @@ function NameAndOrganizationSelector({
                 defaultValue={title}
                 required
                 {...getFormFeedbackForField('name', state)}
+                className="mb-2"
             />
             <div>
-                <Heading4>Legg tavla til i en organisasjon</Heading4>
-                <Paragraph>
-                    Hvis du ikke velger en organisasjon, vil tavla bli lagret
-                    under din private bruker. Det er kun du som kan administrere
-                    private tavler som opprettes.
-                </Paragraph>
+                <Label>Legg til i en organisasjon</Label>
                 <Dropdown
                     items={organizations}
                     label="Dine organisasjoner"
                     selectedItem={selectedOrganization}
                     onChange={setSelectedOrganization}
                     clearable
+                    className="mb-2"
+                    {...getFormFeedbackForField('dropdown', state)}
+                    aria-required="true"
+                    disabled={isPrivate}
                 />
+                <Checkbox
+                    defaultChecked={isPrivate}
```

Comment:
Siden vi skal bruke en state variabel for denne må vi bruke `checked` og ikke `defaultChecked`